odoo/addons/base/models/ir_ui_view.py

```diff
@@ -1671,76 +1673,79 @@ def _validate_module_views(self, module):
 
 
 class ResetViewArchWizard(models.TransientModel):
```

Comment:
double check access on this transient model.
Should be restricted to technical settings groups

Reply:
Ah yes, the access on transient model was introduced after this PR so this one was missing it, well spotted